react/components/CategoriesHighlights/components/CategoryCard.js

```diff
@@ -22,20 +24,22 @@ class CategoryCard extends Component {
 
   render() {
     const { name, image, shape } = this.props
+
     return (
       <div
-        className={`vtex-categories-highlighted__category-${shape}-card shadow-1`}>
+        className={`${categoriesHighlights[shape]}Card shadow-1`}>
```

Comment:
this won't work, you need to apply the `Card` part of the class to the css modules mapping variable. like so
```suggestion
        className={`${categoriesHighlights[`${shape}Card`]}shadow-1`}>
```
you should also do something similar to the rest of the classes